### ISSUE TITLE
FC Live Editor keep injecting

### ISSUE THREAD

**Issue by morote0107** [xAranaktu/FC-24-Live-Editor#293]
when i try to run the game, the live editor keep injecting forever, i try to change the game location to FC 24 but it doesnt work

![image](https://github.com/user-attachments/assets/1b79c083-33d7-42af-abf7-8d9e5e84f7ab)

@xAranaktu

**Comment by AsierBajoJuan** [comment]
That happens to me too. I look in the task manager and I see that the game has started but I don't see it.

**Comment by xAranaktu** [comment]
Only [Live Editor v24.1.1.9](https://www.patreon.com/posts/109025352) work with TU18.2

**Comment by morote0107** [comment]
> Only [Live Editor v24.1.1.9](https://www.patreon.com/posts/109025352) work with TU18.2

Now I have the same problems as other people
![image](https://github.com/user-attachments/assets/242354a5-74c7-4db9-8dfa-5cff2bac852d)
[log_08-08-2024.txt](https://github.com/user-attachments/files/16539944/log_08-08-2024.txt)

**Comment by xAranaktu** [comment]
Probably your injection delay is too high? Change it to 100ms.
If it's at 100ms already then send log file from launcher.

**Comment by morote0107** [comment]
> Probably your injection delay is too high? Change it to 100ms. If it's at 100ms already then send log file from launcher.

its already in 100, im sending log file from launcher.
Im like 4 days trying to make this work jajaj
[log_launcher_08-08-2024.txt](https://github.com/user-attachments/files/16549721/log_launcher_08-08-2024.txt)

**Comment by xAranaktu** [comment]
Are you able to run the game at all? Without live editor?

**Comment by morote0107** [comment]
> Are you able to run the game at all? Without live editor?

yes i can run the game, is very strange because this started to happen when i restarted mi pc from scratch
